### PR TITLE
Issue #15218: Formatted input file for section 7.3.2 Exception Overrides to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218 

https://checkstyle.org/google_style.html#a7.3.2

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule732exceptionoverrides/InputJavadocMethodAndMissingJavadocMethod.java [hunk 1]
```diff
@@ -2,205 +2,149 @@
...
+  }
+
+  String foo6() {
+    return "Fooooooooooooooo" + "oooooooo";
+  }
+
+  // violation below 'Missing a Javadoc comment.'
+  public String foo7() {
+    // making the method
+    // exceed "minLineCount" property limit ( 2 )
```
Comment (rdiachenko) [inline src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule732exceptionoverrides/InputJavadocMethodAndMissingJavadocMethod.java:46]: I don't understand this comment. Why do we need it?

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule732exceptionoverrides/InputJavadocMethodAndMissingJavadocMethod.java:46]: public methods of a public class which are less than 2 lines does not require to have javadoc comment:

https://github.com/checkstyle/checkstyle/blob/0710931bee4815bba1c063c47ee21e1bc04c65c7/src/main/resources/google_checks.xml#L338-L344

We had missing javadoc violation for this method before using the formatter. But formatter formatted it to be a single line method, I added these comments to exceed the limit of 2 lines, so it'll cause violation.

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule732exceptionoverrides/InputJavadocMethodAndMissingJavadocMethod.java:46]: I am good with this.

**Review by romani — APPROVED** [review]
ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**